momentum/utils.py

```diff
@@ -220,7 +220,25 @@ def rebalance_portfolio(universe, ranking_table, config, data):
     ignore_cols = ["TOTAL", "CASH", "PORTFOLIO"]
     sell = []
     zeros = []
-    liquidity_from_sells = 0
+    # cash flow : add new cash to your portfolio or withdraw 
+    print ("Do you want to add cash or withdraw money (y/n)?")
+    answer = input()
+    if answer == "y" :
+        print ("How much? Use negative number if you want to withdraw")
+        new_cash = float(input())
+        if new_cash > 0:
+            print ("You added " + str(new_cash) + " $.")
```

Comment:
Since this is running on Python3, you can use [f-strings formatting](https://realpython.com/python-f-strings/), which makes the code a bit cleaner.

So you'd have something like this:

```
print("You added {new_cash}$.")
```